### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by Falya**

### CODE REVIEW

File: src/methods/clientMethods.mjs [hunk 1]
```diff
@@ -242,6 +240,58 @@ export async function compareOrder(params) {
...
+      date: seance.date,
+      format: seance.format,
+      hallName: seance.hallName,
+      movieInfo: {
+        name: seance.movieName.name,
+        poster: seance.movieName.poster,
+      },
+      movieTheaterInfo: {
+        name: seance.hallId.movieTheaterId.cinemaName,
+        adress: seance.hallId.movieTheaterId.adress,
```
Comment (dzmitry-penda) [inline src/methods/clientMethods.mjs]: address

File: src/methods/clientMethods.mjs [hunk 2]
```diff
@@ -242,6 +240,58 @@ export async function compareOrder(params) {
...
+export async function getUserProfile(user) {
+  const userQuery = User.findById(user._id)
+    .populate({
+      path: 'tickets.seanceId',
+      populate: { path: 'movieName hallId', populate: { path: 'movieTheaterId', populate: { path: 'city' } } },
+    })
+    .select('tickets features -_id');
+  const userInfo = await userQuery;
+
+  const uniqSeances = new Set();
```
Comment (dzmitry-penda) [inline src/methods/clientMethods.mjs]: uniqueSeances

File: src/methods/clientMethods.mjs (hunk 1 above)
Comment (Falya) [reply src/methods/clientMethods.mjs]: fixed

File: src/methods/clientMethods.mjs (hunk 2 above)
Comment (Falya) [reply src/methods/clientMethods.mjs]: fixed

**Pull request merged**